sdk/ai/azure-ai-agents/samples/agents_response_formats/sample_agents_json_schema_response_format.py

```diff
@@ -0,0 +1,101 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+DESCRIPTION:
+    This sample demonstrates how to use agents with JSON schema output format.
+
+USAGE:
+    python sample_agents_json_schema_response_format.py
+
+    Before running the sample:
+
+    pip install azure-ai-projects azure-ai-agents azure-identity pydantic
+
+    Set these environment variables with your own values:
+    1) PROJECT_ENDPOINT - The Azure AI Project endpoint, as found in the Overview
+                          page of your Azure AI Foundry portal.
+    2) MODEL_DEPLOYMENT_NAME - The deployment name of the AI model, as found under the "Name" column in
+       the "Models + endpoints" tab in your Azure AI Foundry project.
+"""
```

Comment:
We have this sample [here](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-agents/samples/agents_files_images_inputs/sample_agents_json_schema.py)